benches/benches/block_target_gas_set/memory.rs

```diff
@@ -0,0 +1,277 @@
+use crate::{
+    utils::{
+        arb_dependent_cost_values,
+        set_full_word,
+    },
+    *,
+};
+
+// ALOC: Allocate memory
+// CFE: Extend call frame
+// CFEI: Extend call frame immediate
+// CFS: Shrink call frame
+// CFSI: Shrink call frame immediate
+// LB: Load byte
+// LW: Load word
+// MCL: Memory clear
+// MCLI: Memory clear immediate
+// MCP: Memory copy
+// MCPI: Memory copy immediate
+// MEQ: Memory equality
+// POPH: Pop a set of high registers from stack
+// POPL: Pop a set of low registers from stack
+// PSHH: Push a set of high registers to stack
+// PSHL: Push a set of low registers to stack
+// SB: Store byte
+// SW: Store word
+pub fn run_memory(group: &mut BenchmarkGroup<WallTime>) {
+    run(
+        "memory/aloc opcode",
+        group,
+        [op::aloc(0x10), op::jmpb(RegId::ZERO, 0)].to_vec(),
```

Comment:
Maybe it is worth allocating non-zero values

Reply:
I just tried a range of values and they were all very similar. But I'll set it to something non-zero for good measure.